### PR TITLE
Fix php stan issues

### PR DESCRIPTION
**Pull request by MarijaIv**
### What is the goal?

Make core library pass phpstan analysis up to level 6.

### Does it affect (changes or update) any sensitive data?

No.

### CODE REVIEW

File: src/BusinessLogic/AdminAPI/AdminAPI.php [hunk 1]
```diff
@@ -47,9 +48,11 @@ public static function get(): object
      */
     public function connection(string $storeId): object
     {
+        // @phpstan-ignore-next-line
```
Comment (m1k3lm) [inline src/BusinessLogic/AdminAPI/AdminAPI.php:51]: Please, could you give some context on the PR message about why we are ignoring this kind of lines and why it is safe to ignore it

File: src/BusinessLogic/Bootstrap/Aspect/Aspects.php [hunk 2]
```diff
@@ -11,15 +11,20 @@
...
  */
+/**
+ * @phpstan-consistent-constructor
+ */
 class Aspects
 {
     /**
      * @var T|null
      */
+    // @phpstan-ignore-next-line
```
Comment (m1k3lm) [inline src/BusinessLogic/Bootstrap/Aspect/Aspects.php:22]: Same for this kind of ignores.
Please, could you give some context on the PR message about why we are ignoring this kind of lines and why it is safe to ignore it

File: src/BusinessLogic/Bootstrap/Aspect/Aspects.php [hunk 3]
```diff
@@ -47,29 +52,49 @@ public function andRun(Aspect $aspect): self
...
      */
     public function beforeEachMethodOfInstance($subject)
     {
         $this->subject = $subject;
         $this->subjectClassName = null;
 
+        /**
+ * @noinspection PhpIncompatibleReturnTypeInspection
+*/
+        // @phpstan-ignore-next-line
```
Comment (m1k3lm) [inline src/BusinessLogic/Bootstrap/Aspect/Aspects.php:67]:
```suggestion
        /**
         * @noinspection PhpIncompatibleReturnTypeInspection
        */
        // @phpstan-ignore-next-line
```

What is the issue here?

File: src/BusinessLogic/Bootstrap/Aspect/Aspects.php [hunk 4]
```diff
@@ -47,29 +52,49 @@ public function andRun(Aspect $aspect): self
...
      * @return T
      */
     public function beforeEachMethodOfService(string $serviceClass)
     {
         $this->subjectClassName = $serviceClass;
         $this->subject = null;
 
+        /**
+ * @noinspection PhpIncompatibleReturnTypeInspection
+*/
```
Comment (m1k3lm) [inline src/BusinessLogic/Bootstrap/Aspect/Aspects.php:85]:
```suggestion
        /**
         * @noinspection PhpIncompatibleReturnTypeInspection
        */
```

**Review by m1k3lm — DISMISSED** [review]

**Review by mescalantea — DISMISSED** [review]
Why stop at level 6 and not move on to level 9? I think that the levels 7, 8 and 9 would help us to prevent a lot of runtime errors, for example the ones related to accessing properties or methods of wrong types or null references

**Pull request merged**